resources/ext/cli/ssl-setup.erb

```diff
@@ -230,10 +230,11 @@ else
   user=<%= EZBake::Config[:user] %>
   group=<%= EZBake::Config[:group] %>
 
+  puppetdb_confdir="/etc/puppetlabs/puppetdb"
   if id -u pe-puppetdb > /dev/null 2>&1; then
-    puppetdb_confdir="/etc/puppetlabs/puppetdb"
+    user=pe-puppetdb
   else
-    puppetdb_confdir="/etc/puppetdb"
+    user=puppetdb
```

Comment:
you shouldn't need to set user, since that's done using the ezbake configs on line 230